src/lib/coda_base/sparse_ledger.ml

```diff
@@ -3,23 +3,19 @@ open Import
 open Snark_params.Tick
 
 module Ledger_hash_binable = struct
-  (* Ledger_hash.t not bin_io *)
-  include Ledger_hash.Stable.V1
+  include Ledger_hash
```

Comment:
`Ledger_hash.t` should not be `bin_io`. If it is, that should be changed. So this change should not compile, in fact.

Reply:
it's not bin_io, so I am going to inline this. None of the inputs, Module, for Sparse_ledger.Make, have a type t that is `bin_io`